python/lsst/sims/GalSimInterface/galSimInterpreter.py

```diff
@@ -644,15 +645,18 @@ def write_checkpoint(self, force=False):
             # the galsim.Images from scratch, given the detector name.
             images = {key: value.array for key, value
                       in self.detectorImages.items()}
+            drawn_objects = self.drawn_objects if object_list is None \
+                            else object_list
             image_state = dict(images=images,
                                rng=self._rng,
-                               drawn_objects=self.drawn_objects,
+                               drawn_objects=drawn_objects,
                                centroid_objects=self.centroid_list)
             with tempfile.NamedTemporaryFile(mode='wb', delete=False,
                                              dir='.') as tmp:
                 pickle.dump(image_state, tmp)
                 tmp.flush()
                 os.fsync(tmp.fileno())
+                os.chmod(tmp.name, 0o660)
```

Comment:
Just checking you wanted this to show up in this PR and it isn't left over from somewhere else.

Reply:
Oops never mind.  I see this was in your PR description.